charts/common/templates/_container.yaml

```diff
@@ -0,0 +1,152 @@
+{{- define "common.container" -}}
+{{- $root := .root }}
+{{- $componentName := .componentName }}
+{{- $containerName := .containerName }}
+{{- $containerValues := .containerValues }}
+{{- $configFiles := .configFiles }}
+- name: {{ $containerName }}
+  image: "{{ $containerValues.image.repository }}:{{ $containerValues.image.tag | default $root.Values.defaultTag }}"
+  imagePullPolicy: {{ default "IfNotPresent" $containerValues.image.pullPolicy | quote }}
+  {{- with $containerValues.command }}
+  command: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with $containerValues.args }}
+  args: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  env:
+    - name: TZ
+      value: {{ default "Europe/Zurich" $root.Values.timezone | quote }}
+  {{- with $containerValues.env  }}
+    {{- if (kindIs "slice" .) }}
+      {{- toYaml . | nindent 4 }}
+    {{- else }}
+      {{- range $key, $value := . }}
+    - name: {{ $key }}
+      value: {{ $value }}
```

Comment:
change this line to `value: {{ $value | quote }}`
At the moment there is a small bug with numbers in env maps, so we can fix it directly with this new release

Reply:
done